test/test_emulation/__init__.py

```diff
@@ -0,0 +1,168 @@
+import fcntl
+import libevdev
+import requests
+import shutil
+import subprocess
+import os
+import pytest
+import threading
+
+from pathlib import Path
+from time import sleep, time
+
+
+class RatbagemuClient(object):
+    def __init__(self, url='http://localhost:8080'):
+        self.url = url
+
+    '''
+    Internal wrappers around the http requests
+    '''
```

Comment:
just a fwiw: AIUI any `'''` is a proper python docstring, which in this case doesn't make sense since anything underscore is pseudo-private in python. So for comments that merely explain structure of the code, use `#` insteadl.